stories/topics/ref-gcp-install-permissions.adoc

```diff
@@ -2,87 +2,29 @@
 
 = Permissions
 
-Your GCP account must have the following permissions to successfully create an {PlatformNameShort} deployment on _{GCP}_.
+Your GCP account must have the following _Identity and Access Management_ (IAM) permissions to successfully create and manage {PlatformNameShort} deployments as well as the resources described in xref:con-gcp-application-architecture[Application architecture].
 
+Your GCP account must also be licensed to deploy {AAPonAWS}.
 
-* Cloud SQL Permissions
```

Comment:
Is there a reason you want to remove the list of all the individual permissions?

Reply:
Because it wasnt actually valid as far as I can tell.
In the UI, when you are deploying an offer, the only Service accounts which appear in the dropdown from what could see, are the ones that meet this new minimum permission list.

I tried to manually create a role with the removed noted permissions, and add that to a SA, but this SA did not show up in the marketplace form where the user can select an SA. If a user creates an SA, it will use the new added roles here.

Reply:
I am not sure the origin of this permissions table but wonder if it was based on looking at the aap-user and seeing all the excess permissions that were listed.  
<img width="1430" alt="image" src="https://github.com/ansible/aap-docs/assets/32707829/37ee4fb1-8993-4642-b3c0-918446019eab">

Not sure if @itdove or @leena-jawale knows more about this or if Robin was the one who created this in the old GCP project and there was a specific reason for listing a subset of the permissions for the roles

Reply:
The only SAs that show up in the drop down are the ones that meet the list shown above which contain the noted roles - 
![image](https://github.com/ansible/aap-docs/assets/52431213/bc4dd07b-08e5-4d1e-bd23-14c25a31b477).

I created this [role](https://console.cloud.google.com/iam-admin/roles/details/projects%3Cagcp-001-dev%3Croles%3CCustomRole401?project=agcp-001-dev), named - `Install - Existing VPC min permissions`, which has that full list of 70 permissions. 
However if you assign this role to an SA, it still can't be used to deploy from the UI from what I've seen in my attempts. This likely would work when deploying from the CLI though.

Reply:
Yes, I looked at this an hour or so ago and see the same.  But before deleting this part from the doc I wanted to be sure there was not some other reason it was added in the first place and that we were not aware of.

Reply:
I also validated it and it's strange that we can't see that SA in the dropdown. But I think we should doc what customers can see in the UI and whatever we have validated. The changes that Zack did, look good to me.

Reply:
@prat98 Can you weigh in here when you get a chance?

Reply:
I believe @zkayyali812 is correct, the min permissions documented before work if you wanted to create a deployment from the CLI. Those permissions were essentially generated by taking a look at the excess permissions granted to the aap-user SA. I think for a SA to show up in marketplace it only checks if the SA has the specific GCP roles attached to it. Even if an SA can actually create a deployment with a bunch of individual permissions, it will not show up if the specified GCP roles are not attached to the SA. Because we currently doc using the marketplace, @zkayyali812 changes make more sense than specifying every single permission required.